yarn.lock

```diff
@@ -2606,6 +2615,22 @@ boolbase@^1.0.0, boolbase@~1.0.0:
   resolved "https://registry.yarnpkg.com/boolbase/-/boolbase-1.0.0.tgz#68dff5fbe60c51eb37725ea9e3ed310dcc1e776e"
   integrity sha1-aN/1++YMUes3cl6p4+0xDcwed24=
 
+bootstrap-vue@^2.1.0:
```

Comment:
looks like now there are two versions of bootstrap , suggest to pick ( bump to ) one

Reply:
bootstrap (no vue) is stil necessary to be compiled with our custom color variables.
Bumped version to match bootstrap-vue.

cleaned import statements in components